Modules/Microsoft365DSC/DSCResources/MSFT_SPOSharingSettings/MSFT_SPOSharingSettings.psm1

```diff
@@ -93,7 +93,7 @@ function Get-TargetResource
 
         [Parameter()]
         [System.String]
-        [ValidateSet("None", "View", "Edit")]
+        [ValidateSet("View", "Edit")]
```

Comment:
Wouldn't this be kind of a breaking change, as we remove one possible value (even though it did not work)?

Reply:
Yes, agreed with Andi. Let's park it for the moment.

Reply:
Actually, I think we should continue with this change, but only internally. We could keep the wrong validateset but still update the inner parts of the resource to handle NONE in the correct way.

Reply:
So I'll restore None to validate set. I handle the None on get but will add logic on set to convert None to Edit

Reply:
Can we also print a warning that the value is deprecated if anyone uses the value?

Reply:
ok added a warning message, added None back to validateset and handled within app